src/neo/SmartContract/Native/Tokens/NeoToken.Economic.cs

```diff
@@ -0,0 +1,154 @@
+#pragma warning disable IDE0051
+
+using Neo.Cryptography.ECC;
+using Neo.Ledger;
+using Neo.Persistence;
+using System;
+using System.Numerics;
+
+namespace Neo.SmartContract.Native.Tokens
+{
+    public partial class NeoToken
+    {
+        private const byte Prefix_GasPerBlock = 17;
+        private const byte Prefix_NeoHoldersRewardRatio = 73;
+        private const byte Prefix_CommitteeRewardRatio = 19;
+        private const byte Prefix_VotersRewardRatio = 67;
+        private const byte Prefix_VoterRewardPerCommittee = 23;
+        private const byte Prefix_HolderRewardPerBlock = 57;
+
+        [ContractMethod(0_05000000, CallFlags.AllowModifyStates)]
+        private bool SetGasPerBlock(ApplicationEngine engine, BigInteger gasPerBlock)
+        {
+            if (gasPerBlock < 0 || gasPerBlock > 8 * GAS.Factor) return false;
+            if (!CheckCommitteeWitness(engine)) return false;
+            StorageItem item = engine.Snapshot.Storages.GetAndChange(CreateStorageKey(Prefix_GasPerBlock));
+            item.Value = gasPerBlock.ToByteArray();
+            return true;
+        }
+
+        [ContractMethod(0_05000000, CallFlags.AllowModifyStates)]
+        private bool SetRewardRatio(ApplicationEngine engine, byte neoHoldersRewardRatio, byte committeesRewardRatio, byte votersRewardRatio)
+        {
+            if (checked(neoHoldersRewardRatio + committeesRewardRatio + votersRewardRatio) != 100) return false;
+            if (!CheckCommitteeWitness(engine)) return false;
+            StorageItem holderItem = engine.Snapshot.Storages.GetAndChange(CreateStorageKey(Prefix_NeoHoldersRewardRatio));
```

Comment:
If we make only one call to the storage, it could be faster, serialize a class with these three values